admin/class-openedx-commerce-admin.php

```diff
@@ -470,11 +470,11 @@ public function items_enrollment_request( $courses, $order_id, $billing_email, $
 			$action      = 'enrollment_process';
 
 			$enrollment_arr = array(
-				'enrollment_course_id'    => $course_id,
-				'enrollment_email'        => $billing_email,
-				'enrollment_mode'         => $course_mode,
-				'enrollment_request_type' => $request_type,
-				'enrollment_order_id'     => $order_id,
+				'openedx_enrollment_course_id' => $course_id,
+				'openedx_enrollment_email'     => $billing_email,
+				'openedx_enrollment_mode'      => $course_mode,
+				'enrollment_request_type'      => $request_type,
```

Comment:
Should these other two variables also change for consistency?

Reply:
Yes, changed!! Thanks.